src/login/components/LoginFormWalletFile/LoginFormWalletFile.js

```diff
@@ -76,19 +118,31 @@ export default class LoginFormWalletFile extends React.Component {
 
   handleSubmit = (event) => {
     event.preventDefault();
-    this.props.onLogin({ wif: this.props.wif });
+    const { encryptedWIF, passphrase = '', onLogin } = this.props;
```

Comment:
You shouldn't need to assign `passphrase = ''`.  If it's not a string here, there's a bug.